packages/flutter/lib/src/widgets/app.dart

```diff
@@ -12,6 +12,8 @@ library;
 
 import 'dart:collection' show HashMap;
 
+// ignore: unused_import, required for the Widget Inspector's on-device button.
+import 'package:flutter/cupertino.dart' show CupertinoIcons;
```

Comment:
Importing cupertino into the widgets library is layer violation and I don't think the analyzer script is going to like this.

Reply:
Uhm I don't see the symbols imported from `CupertinoIcons` being used anywhere. Does it work without this import?

Reply:
Unfortunately no, that's why I added this note to the PR:

> For some reason, adding the import to packages/flutter/lib/src/material/app.dart and packages/flutter/lib/src/cupertino/app.dart did not resolve the issue. That's why I've added it to lib/src/widgets/app.dart, even though it's not actually used in that file.

I first tried adding the import to where we are actually using the icon (in the two files mentioned above) however despite that change I was still seeing the `[?]` instead of the icon. I do not understand why though 😕

Reply:
You tested in debug mode (so no icon tree shaking) right? In that case I don't know off the top of my head. I'll try to debug this.